src/create_atoms.cpp

```diff
@@ -303,7 +303,7 @@ void CreateAtoms::command(int narg, char **arg)
     if (onemol->xflag == 0) error->all(FLERR, "Create_atoms molecule must have coordinates");
     if (onemol->typeflag == 0) error->all(FLERR, "Create_atoms molecule must have atom types");
     if (ntype + onemol->ntypes <= 0 || ntype + onemol->ntypes > atom->ntypes)
-      error->all(FLERR, "Invalid atom type in create_atoms mol command");
+      error->all(FLERR, "Invalid atom type {} in create_atoms mol command (onemol->ntypes {} atom->ntypes {})", ntype, onemol->ntypes, atom->ntypes);
```

Comment:
```suggestion
      error->all(FLERR, "Invalid atom type {} in create_atoms mol command", ntype + onemol->ntypes);
```
For molecules, ntype is only the offset added to the atom types in the molecule file.